### ISSUE TITLE
"celery or roblox app needs update"

### ISSUE THREAD

**Issue by dahoodsucks** [0xVienna/Celery#12]
when i update roblox app, it doesnt work. also where is "Library"

**Comment by dahoodsucks** [comment]
also i just installed celery and it says celery/roblox needs an update.

**Comment by 0xVienna** [comment]
yeah make sure you use Celery Installer because it installs the latest version of celery. Just update

**Comment by dahoodsucks** [comment]
> yeah make sure you use Celery Installer because it installs the latest version of celery. Just update

It says up to date.

**Comment by anhtlauncher** [comment]
> > yeah make sure you use Celery Installer because it installs the latest version of celery. Just update
> 
> It says up to date.

same

**Comment by 0xVienna** [comment]
Up to date just attach and run ur script

**Comment by 0xVienna** [comment]
Run as admin the celery

**Comment by Noelle121-sys** [comment]
> Run as admin the celery

![image_2023-05-12_160913468](https://github.com/0xVienna/Celery/assets/133325652/af8633f9-6fda-4d0c-804e-0033ed50f21d)

still not working

**Comment by 0xVienna** [comment]
Oh you have to open Microsoft store and press update

**Comment by Noelle121-sys** [comment]
![image_2023-05-12_163605777](https://github.com/0xVienna/Celery/assets/133325652/bc9f51ca-29cb-4c8d-a66b-b28823e7a578)

i already update but it still tell me to update

**Comment by 0xVienna** [comment]
Um delete the whole celery and reinstall WITH THE NEW ONE

**Comment by Noelle121-sys** [comment]
> Um delete the whole celery and reinstall WITH THE NEW ONE

done, but still the same, but its fine now i give up, thanks for your help anyways.

**Comment by 0xVienna** [comment]
ok sorry you can use fluxus
